### PR TITLE
Working Branch

### PR DESCRIPTION
**Pull request by Amita-Roy**
Hi, we have cloned this project from [https://www.newsweek.com/](url). The main focus for this project was to work with Bootstrap.

- We have added Bootstrap as a npm package
- To work with bootstrap variables we are using main.scss file which is later compiling to style.css.
- Our customised style are in **SCSS/main.scss** file and we have corrected all the stylelint error for our main.scss file.
- Html file is also validated

### CODE REVIEW

**Comment by Amita-Roy** [comment]
we have made all the changes required in the review and we have added Bootstrap as per the link provided in the odin project. Please refer the screenshot.

bootstrap

**Comment by Amita-Roy** [comment]
Thank you for the reviews, we have made all the required changes.

**Comment by bertil291utn** [comment]
**Status:** Changes requested 🔴  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.


You have achieved an amazing job with your README file, commit messages and your GitHub Flow, however in order to accomplish as the actual page, you required to do these changes, listed below: 

**Small screen**

- [ ] **Footer section.-** Copyright section should be under Newsweek logo 

| ACTUAL WEB PAGE                                              | YOURS                                                        |
| ------------------------------------------------------------ | ------------------------------------------------------------ |
| image | image |

- [ ] **Category tag .-** Each article has a tag call category, try to adjust each element width on each article on all sections.

| ACTUAL WEB PAGE                                              | YOURS                                                        |
| ------------------------------------------------------------ | ------------------------------------------------------------ |
| image | image |

- [ ] **Category tag .-** Each article has a tag call category, **IN THE MAGAZINE** section, try to work with a red background and red color font

| ACTUAL WEB PAGE                                              | YOURS                                                        |
| ------------------------------------------------------------ | ------------------------------------------------------------ |
| image | image |

**Medium screen**

- [ ] Reduce the size of each avatar on the **THE DEBATE** section

image

- [ ] Use `object-fit: cover;` to avoid stretching images on **IN THE MAGAZINE SECTION**. Same for a large screen in this section

image

- [ ]  After the editor's pick section, try to fill the blue square should be the same height as your text. Remember to work on each section where you have a similar situation. Same for a large screen in  this section

image


Guys work in the requested changes above and request another review.
Good luck. 
<hr>

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Comment by eypsrcnuygr** [comment]
Copyright section under the logo for the.
tags are adjusted.
In the magazine section background and color changed.
The debate section images reduced.
In the magazine section, images don't stretch now.
Editors pick section, images having the full height except for the first ones.

**Comment by eypsrcnuygr** [comment]
In the magazine section, the images are bigger for the xl screens.

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,7409 @@
...
+  vertical-align: bottom !important; }
+
+.align-text-bottom {
+  vertical-align: text-bottom !important; }
+
+.align-text-top {
+  vertical-align: text-top !important; }
+
+.bg-primary {
+  background-color: red !important; }
```
[the remark this answers is not in the document]
Comment (Amita-Roy) [reply styles/styles.css:4722]: Our customised styles are in main.scss. The styles.css is compiling from bootstrap which is nothing to do with our customised styles.

**Comment by Amita-Roy** [comment]
We removed all the inline style from html.
Removed all the unused classes from html.

Our customised styles are in main.scss. The styles.css is compiling from bootstrap which is nothing to do with our customised styles.

**Comment by kitpao** [comment]
# 👋 Hello Team,

Nice work so far! 👏 You are almost done! 

🔍You only missed these details: 

- [ ] Your Subscribe and Newsletter titles are still in black instead of red

- [ ] Please, position this image correctly in the mobile version (centered)

Do this and you will be done!


___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Pull request merged**